src/pages/ExportTransactions/index.tsx

```diff
@@ -83,6 +83,8 @@ const ExportTransactions = () => {
     )
   }
 
+  // disable download button when downloading
+  const [downloading, setDownloading] = useState(false)
```

Comment:
Use `is~` to indicate that it's a boolean

Reply:
updated: [79d2ef2](https://github.com/Magickbase/ckb-explorer-frontend/pull/73/commits/79d2ef28604670bada79de9edd9e9828a61ca01c)